utils/createResponseObj.js

```diff
@@ -7,12 +7,15 @@
 } */
 
 const createResponseObj = (data, metadata, statusCode, success = true) => {
+  if (data.password) {
+    delete data.password;
+  }
```

Comment:
I think it is wrong to delete the password in this function, but we'd better wait and see what others think

Reply:
I think it's okay to have this check here, just verify whether the data exists or not.
```
if (data?.password) {
    delete data.password;
}
```